st/clitests/integrity.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+import json
+import uuid
+import random
+import argparse
+
+import plumbum
```

Comment:
It depends on how this test would be executed. If it would be a part of CI then the answer is likely "yes", but it's not now.
If it's only for a manual execution then it's for you (or someone else from S3 server team) to decide.

Please let me know about the decision. Or I could spend more time to write whatever is present in that library already and use `subprocess` directly.

Reply:
checks are added to jenkins_buildd